toonz/sources/toonz/mainwindow.cpp

```diff
@@ -2344,9 +2378,9 @@ void RecentFiles::saveRecentFiles() {
 
 QList<QString> RecentFiles::getFilesNameList(FileType fileType) {
   QList<QString> files =
-      (fileType == Scene)
-          ? m_recentScenes
-          : (fileType == Level) ? m_recentLevels : m_recentFlipbookImages;
+      (fileType == Scene) ? m_recentScenes : (fileType == Level)
```

Comment:
This is an example of #715 - These are the results of running clang-format using the beautification scripts.